### PR TITLE
Fixed a typo in INSTALL.md

### PR DESCRIPTION
**Pull request by aka001**

### CODE REVIEW

**Comment by nehaljwani** [comment]
Confirmed from http://pkgs.org/search/build-essential
But there is a small problem. RHEL and derivatives don't have any package by the name "build-essentials". We should remove that. @aka001 Could you please fix that and send a PR again?

File: INSTALL.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ Following are the steps to install the above dependencies for few major distribu
 ### For CentOS, RHEL, Fedora
 
 ```
-sudo yum install python python-pip build-essentials python-devel
+sudo yum install python python-pip build-essential python-devel
```
Comment (aka001) [inline INSTALL.md]: @nehaljwani sir I have edited the package name here too. Is this what you were mentioning earlier ? 
Kindly clarify the same.

Comment (nehaljwani) [inline INSTALL.md]: There is no package by the name 'build-essential' in centos/rhel/fedora.

Comment (aka001) [inline INSTALL.md]: @nehaljwani sir, I have updated the pull request and squashed the commits.

**Pull request merged**